### ISSUE TITLE
Crash

### ISSUE THREAD

**Issue by iamajijul** [Giphy/giphy-android-sdk#133]
We are facing one crash when we are trying to select any GIF from giphy dialog. Below are the step to re produce : 
1. Open giphy dialog
2. press the filter button(Below the giphy search bar) and quickly select any GIF.
<img width="544" alt="Screen Shot 2021-06-06 at 11 57 14 AM" src="https://user-images.githubusercontent.com/23631066/120917160-9f156f00-c6be-11eb-8820-a9267e62de4e.png">
3. Observe the crash below crash

java.lang.IndexOutOfBoundsException: Index: 2
        at java.util.Collections$EmptyList.get(Collections.java:4511)
        at androidx.recyclerview.widget.ListAdapter.getItem(ListAdapter.java:150)
        at com.giphy.sdk.ui.universallist.SmartGridAdapter.access$getItem(SmartGridAdapter.kt:21)
        at com.giphy.sdk.ui.universallist.SmartGridAdapter$onBindViewHolder$1.onClick(SmartGridAdapter.kt:59)
        at android.view.View.performClick(View.java:8160)
        at android.view.View.performClickInternal(View.java:8137)
        at android.view.View.access$3700(View.java:888)
        at android.view.View$PerformClick.run(View.java:30250)
        at android.os.Handler.handleCallback(Handler.java:938)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:246)
        at android.app.ActivityThread.main(ActivityThread.java:8425)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)

**Comment by ALexanderLonsky** [comment]
@iamajijul thanks for flagging this and steps to reproduce.
Please try if this version works for you: 
`implementation "com.giphy.sdk:ui:2.1.0-grid-crash-fixed"`

For that, you'll need to declare the Maven staging repository:
```
    repositories {
         maven("https://oss.sonatype.org/content/repositories/staging")
```

**Comment by ALexanderLonsky** [comment]
Must be fixed in [v2.1.1](https://github.com/Giphy/giphy-android-sdk/releases/tag/v2.1.1)
